src/realtime/websocket/extract-note-id-from-path.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2022 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+/**
+ * Extracts the note id from the given realtime URL path.
+ *
+ * @param realtimePath The path from which the note id should be extracted
+ * @return The extracted note id
+ * @throws Error if the given string isn't a valid realtime URL path
+ */
+export function extractNoteIdFromPath(realtimePath: string): string {
```

Comment:
Why is this file named `extract-note-id-from-path.ts` and not `util.ts`? And why is this function not part of `websocket.gateway.ts`?

Reply:
Why should it be called util? It exports one function.

Easier to test and less cluttering in the gateway file.

Reply:
I think it's inconsistent. There are many other static functions in the gateway file which are not extracted.